invenio_communities/views/communities.py

```diff
@@ -235,3 +235,34 @@ def invitations(community=None, pid_value=None):
         },
         permissions=permissions,
     )
+
+
+@pass_community
+def invitation_details(community=None, pid_value=None, invitation_pid_value=None):
+    invitation = current_requests_service.read(
+        id_=invitation_pid_value, identity=g.identity
+    )
+    query_config = dict(
+        size=current_app.config['REQUESTS_TIMELINE_PAGE_SIZE']
+    )
+
+    permissions = community.has_permissions_to(
+        ['update', 'read', 'search_requests', 'search_invites']
+    )
+    if not permissions['can_search_invites']:
+        raise PermissionDeniedError()
+
+    return render_template(
+        "invenio_communities/details/members/invitation_details.html",
+        community=community.to_dict(),
+        types={
+            "organization": "Organization",
+            "event": "Event",
+            "topic": "Topic",
+            "project": "Project"
+        },
+        permissions=permissions,
+        request=invitation.to_dict(),
+        default_query_config=query_config,
+        custom_base_template="invenio_communities/details/members/base.html"
```

Comment:
I suggest to set this variable in the template, it will be easier to override in case of need